namerd/core/src/main/scala/io/buoyant/namerd/DtabStoreInitializer.scala

```diff
@@ -22,5 +23,9 @@ abstract class DtabStoreConfig extends PolymorphicConfig {
   def disabled = experimentalRequired && !_experimentalEnabled.contains(true)
 
   @JsonIgnore
+  def mkDtabStore(params: Stack.Params): DtabStore = mkDtabStore
+
+  @JsonIgnore
+  @Deprecated
```

Comment:
Rather than keeping this deprecated method, how would you feel about just updating all of the existing DtabStore implementations?  There are only a small handful in the codebase and it should be a very mechanical change.

Reply:
Won't this break all custom plugins that linked users might have?

I prepared this in haste, and I do see now, that fix is not optimal, as new plugin would need to implement both methods.
I can fix that and update all plugins that come with linked, but I believe it's better to keep the old method anyway.

Reply:
I won't insist though if you decide to remove the old method :)